src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java

```diff
@@ -405,70 +404,6 @@ void whereLastAccessedIsNullUseCreatedAtAndInProgressStatus() {
         assertEquals(EXPIRED, tod.getTransformedMedia().getMediaRequest().getStatus());
     }
 
-    @Test
-    void whereLastAccessedWasOneDayAgo() {
-        whereLastAccessedWasOneDayAgoSupport(false, false);
-    }
-
-    @Test
-    void whereLastAccessedWasOneDayAgoButMediaIsHidden() {
-        whereLastAccessedWasOneDayAgoSupport(true, false);
-    }
-
-    @Test
-    void whereLastAccessedWasOneDayAgoButMediaIsDeleted() {
-        whereLastAccessedWasOneDayAgoSupport(false, true);
-    }
-
-    @Test
-    void whereLastAccessedWasOneDayAgoButMediaIsHiddenAndDeleted() {
-        whereLastAccessedWasOneDayAgoSupport(true, true);
-    }
-
-    void whereLastAccessedWasOneDayAgoSupport(boolean isHidden, boolean isDeleted) {
-        HearingEntity hearing = dartsDatabase.createHearing(
-            "NEWCASTLE",
-            "Int Test Courtroom 2",
-            "2",
-            HEARING_DATE
-        );
-
-
-        // Last accessed on a 2023-10-20 friday
-        MediaRequestEntity currentMediaRequest = MediaRequestTestData.createCurrentMediaRequest(
-            hearing,
-            requestor,
-            OffsetDateTime.parse("2023-06-26T13:00:00Z"),
-            OffsetDateTime.parse("2023-06-26T13:45:00Z"),
-            AudioRequestType.DOWNLOAD, COMPLETED
-        );
-        currentMediaRequest = dartsDatabase.save(currentMediaRequest);
-
-
-        MediaEntity mediaEntity =
-            MediaTestData.createMediaWith(hearing.getCourtroom(), currentMediaRequest.getStartTime(), currentMediaRequest.getEndTime(), 1);
-        mediaEntity.setHidden(isHidden);
-        mediaEntity.setDeleted(isDeleted);
-        dartsDatabase.save(mediaEntity);
-        hearing.getMediaList().add(mediaEntity);
-        dartsDatabase.save(hearing);
-        TransientObjectDirectoryEntity notMarkedForDeletion = createStoredTransientDirectory(
-            currentMediaRequest,
-            OffsetDateTime.parse("2023-10-22T13:45:00Z")
-        );
-
-        //setting clock to 2023-10-23 on a monday
-        when(currentTimeHelper.currentOffsetDateTime()).thenReturn(OffsetDateTime.of(2023, 10, 23, 22, 0, 0, 0, ZoneOffset.UTC));
-
-        if (isHidden || isDeleted) {
-            assertEquals(1, outboundAudioDeleterProcessor.markForDeletion().size());
-            assertTransientObjectDirectoryStateChanged(notMarkedForDeletion.getId());
-        } else {
-            assertEquals(0, outboundAudioDeleterProcessor.markForDeletion().size());
-            assertTransientObjectDirectoryStateNotChanged(notMarkedForDeletion.getId());
-        }
-    }
-
```

Comment:
Why have these been deleted?

Reply:
Merge issue by the looks of it.

Reply:
These tests exist in master so I would have expected these to still exist in this branch?